devel/tortoisehg/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           app 1.0
 PortGroup           gitlab 1.0
 
 gitlab.instance     https://foss.heptapod.net
-gitlab.setup        mercurial/tortoisehg thg 5.8rc0
+gitlab.setup        mercurial/tortoisehg thg 5.8.1
```

Comment:
Just to make sure: does this count as version increase?

Reply:
Not sure what you mean? IIRC thg 5.8 wasn't out yet when I bumped Mercurial, so I had to use the rc, although that's generally a bad practice. Any 5.8.x _should_ be compatible, but I assume you tested that?

Reply:
(That question wasn't about why you changed anything. Yes, ideally we would want to avoid switching to RC, but here my question is different.)

Assuming someone already had 5.8rc0 installed: does MacPorts see 5.8.1 as an update compared to 5.8rc0 or not?

I didn't test whether tortoisehg 5.8rc0 was compatible with mercurial 5.8.1, I just remember that thg was very often broken when mercurial was updated without updating thg as well. Maybe that was true for major version changes only, but since the update is there, there's no reason not to update it to a totally compatible version.

Reply:
I just ran a similar check; Mercurial 5.9 is actually released, it appears — at least to PyPI. I had an earlier, uncommited version of Mercurial 5.9rc0 installed, and it didn't upgrade it. In this case, we should probably either upgrade Mercurial to 5.9 now, or wait until TortoiseHg 5.9 is released, and upgrade _then_.